src/smsCommandFields/FieldDataElementWithCategoryOptionCombo.js

```diff
@@ -42,32 +42,28 @@ export const FieldDataElementWithCategoryOptionCombo = ({
 
                     return (
                         <>
-                            <FormRow>
-                                <InputFieldFF
-                                    {...rest}
-                                    input={input}
-                                    meta={meta}
-                                    error={!!meta.error}
-                                />
-                            </FormRow>
+                            <InputFieldFF
+                                {...rest}
+                                className={styles.codeField}
+                                input={input}
+                                meta={meta}
+                                error={!!meta.error}
+                            />
 
-                            {code && (
-                                <>
-                                    <Button
-                                        onClick={() => setShowFormula(true)}
-                                    >
-                                        {i18n.t('Change formula')}
-                                    </Button>
+                            <FieldDataElementWithCategoryOptionComboAddFormulaButton
```

Comment:
Names like these are too long imo. They stop being easy to parse and start being easy to misread.